sdk/tests/tutorials/properties/test_transaction_properties.py

```diff
@@ -0,0 +1,144 @@
+# import general python packages
+import unittest
+import json
+from datetime import datetime
+import pytz
+import logging
+
+# import lusid specific packages
+import lusid
+import lusid.models as models
+from utilities import InstrumentLoader
+from utilities import TestDataUtilities
+
+
+class TransactionProperty(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # setup logging configuration
+        logging.basicConfig(level=logging.INFO)
```

Comment:
try

```
root_logger = logging.getLogger()
root_logger.setLevel(logging.INFO)
```

Reply:
Yep just did in the last PR. Looking more promising, I think.

Reply:
Passed! ✔️